### DISCUSSION TITLE
Allow custom installation directory

### DISCUSSION THREAD

**Discussion by IcyAlmond** [LinSoftWin/Photoshop-CC2022-Linux#46] [Ideas]
Allow setting a custom installation directory. I tried to do something like that here: https://github.com/Latrolage/Photoshop-CC2021-Linux/blob/main/installer.sh feel free to just copy paste sections.

It saves the location of the prefix to `$XDG_CONFIG_HOME/adobelinux/photoshoplocation` and if $XDG_CONFIG_HOME doesn't exist it saves it to `~/.config/adobelinux/photoshoplocation` I made the uninstaller script in the same repository read from those locations, but it may need more testing.

**Comment by MiMillieuh** [comment]
If you use the scripts in release or in the scripts folder,  you can already use custom folder.
